docs/intro/contributing.txt

```diff
@@ -68,6 +68,16 @@ It contains lots of great information and is a must read for anyone who'd like
 to become a regular contributor to Django. If you've got questions, it's
 probably got the answers.
 
+.. note::
+
+    All of the instructions in this tutorial assumes you are using Python 3. You
+    might want to install it if you don't have it already. The easiest way is
+    through a package manager, by running e.g. ``apt-get install python3`` or
```

Comment:
We could borrow the language we use on topics/install: "Get the latest version of Python at https://www.python.org/download/ or with your operating system’s package manager."